lib/pages/people/view/person_view.dart

```diff
@@ -0,0 +1,124 @@
+                        )
+                            : CircleAvatar(
+                          radius: 50,
+                          child: Icon(
+                            Icons.person,
+                            size: 50,
+                          ),
+                        ),
+                      ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          RichText(
+                            maxLines: 2,
+                            softWrap: true,
+                            text: TextSpan(children: [
+                              WidgetSpan(
+                                child: Padding(
+                                  padding: const EdgeInsets.only(right: 4.0),
+                                  child: Icon(Icons.email),
+                                )
+                              ),
+                              TextSpan(text: person.email ?? '-')
+                            ]),
+                          ),
+                          SizedBox(height: 8),
+                          RichText(
+                            maxLines: 2,
+                            softWrap: true,
+                            text: TextSpan(children: [
+                              WidgetSpan(
+                                child: Padding(
+                                  padding: const EdgeInsets.only(right: 4.0),
+                                  child: Icon(Icons.phone),
+                                )
+                              ),
+                              TextSpan(text: person.phone ?? '-')
+                            ]),
+                          ),
+                          SizedBox(height: 8),
+                          RichText(
+                            maxLines: 2,
+                            softWrap: true,
+                            text: TextSpan(children: [
+                              WidgetSpan(
+                                child: Padding(
+                                  padding: const EdgeInsets.only(right: 4.0),
+                                  child: Icon(Icons.location_on),
+                                )
+                              ),
+                              TextSpan(text: person.office ?? '-')
+                            ]),
+                          ),
+                          SizedBox(height: 8),
+                          RichText(
+                            maxLines: 2,
+                            softWrap: true,
+                            text: TextSpan(children: [
+                              WidgetSpan(
+                                  child: Padding(
+                                    padding: const EdgeInsets.only(right: 4.0),
+                                    child: Icon(Icons.work),
+                                  )
+                              ),
+                              TextSpan(text: person.position ?? '-')
```

Comment:
I was gonna write about the comma again but the fact I have to do that 4 times pointed out that this is basically the same code x4. Wouldn't it be better to just have a separate, stateless widget (`IconText` or something of the sort) that takes an `IconData` and a `String`, and displays the text next to the icon?

Reply:
For the record, since I know this currently still overflows, having a single widget will also mean that when we do find a fix for the overflow, we only need to write it once.

Reply:
Update: I did this myself in the meantime, for another functionality. Will merge it soon, but you can check it out here: https://github.com/acs-upb-mobile/acs-upb-mobile/commit/095a256c10b3bae0229f859ef9c57d03c4021296